examples/ScaleFactors.ipynb

```diff
@@ -0,0 +1,974 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f0368027d50>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "slopes, slopes_CI = \\\n",
+    "    find_effSDbeam(interferometer_hdu, singledish_hdu, largest_scale * u.arcsec, lowresfwhms,\n",
+    "                   verbose=True)\n",
```

Comment:
Why are these "slopes"?    And why are you looking for slope = 0?  I haven't developed an intuition for this yet, so it probably will become obvious, but it would be good to target this notebook at beginners.